lib/printer.js

```diff
@@ -954,6 +1006,9 @@ function genericPrintNoParens(path, options, print) {
     case "Specifier":
     case "NamedSpecifier":
     case "Comment": // Supertype of Block and Line.
+    case "MemberTypeAnnotation": // Flow
+    case "TupleTypeAnnotation": // Flow
+    case "Type": // Flow
```

Comment:
@cpojer, I may be missing something but is there a particular reason why 'TupleTypeAnnotation' is meant to cause an unprintable error? 

I have been hitting this error:

```javascript
Error: unprintable type: "TupleTypeAnnotation"
    at genericPrintNoParens 
```